lona/html/data_binding/select.py

```diff
@@ -1,81 +1,23 @@
-from copy import copy
-
-from lona.html.nodes import SelectNode, OptionNode, Widget
 from lona.events.event_types import CHANGE
+from lona.html.node import Node
 
 
-class Select(Widget):
-    ID_LIST = []
-    CLASS_LIST = []
-    STYLE = {}
-    ATTRIBUTES = {}
+class Option(Node):
+    TAG_NAME = 'option'
 
-    def __init__(self, values=[], disabled=False, bubble_up=False,
-                 **select_kwargs):
 
-        self.bubble_up = bubble_up
+class Select(Node):
```

Comment:
Don't you want to move it to `lona/html/nodes.py`?

Reply:
Select is no widget anymore, but it keeps implementing data-binding. Therefore i would let it stay in ``lona.html.data_binding``.

Reply:
What does it mean "data-binding"? It is not clear for me.

Reply:
If you change the value of a ``TextInput`` in your browser, the data gets changed in the frontend and the backend. If you change it using Python ``text_input.value = 'foo'``, it also changes in the frontend and the backend. Thats called data-binding.

Reply:
Am I right, that it is not enough to execute `text_input.value = 'foo'` on the server side? I also have to call `view.show()` to send patches to the browser.

Reply:
Yes thats right

Reply:
Seems now I've got the point.
Any `Node` has one way "binding" (changes from server goes to the browser). For example `node.class_list.add("foo")`.
But only nodes in `data_binding` package have two way binding for some attributes (for now it is `.value`).